View2_A/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="match_parent"
+        android:layout_height="100dp"
+        app:srcCompat="@drawable/institute"
+        tools:layout_editor_absoluteX="-16dp"
+        tools:layout_editor_absoluteY="5dp" />
```

Comment:
Do not make use of `layout_editor_absoluteY` or `layout_editor_absoluteX`. Instead place the view appropriately with appropriate margin/padding.